src/views/components/probe-single-card.html

```
@@ -1,49 +1,53 @@
+					>
+			{{/if}}
+			<span class="probe-single-card_network_name">{{ location.network }}</span>
+		</a>
 	</div>
-	<div class="probe-single-card_tag">
-		{{#each tags: index}}
-			{{#if index < maxTagCount-1 || (index === maxTagCount-1 && tags.length <= maxTagCount)}}
-				<span class="probe-single-card_tag_item" as-tooltip="this">
-					{{this}}
+
+	{{#if tags && tags.length > 0}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)